torch/_dynamo/variables/builder.py

```diff
@@ -231,6 +231,10 @@ def make_guards(self, *guards):
     def _wrap(self, value):
         from ..comptime import comptime
 
+        if callable(value):
+            if hasattr(value, "_dynamo_forbidden") and value._dynamo_forbidden:
+                raise AssertionError(f"Attempt to trace forbidden callable {value}")
```

Comment:
This seems awkward? It suggests that if I merely reference a forbidden function such that it gets a tracker, but I don't call it, it will still result in an error. That seems bad.

Reply:
Moved